tests/config/dynamic.tests.yaml

```diff
@@ -109,6 +109,40 @@ cname:
       - pool: iad
   type: CNAME
   value: target.unit.tests.
+dname:
+  dynamic:
```

Comment:
Any idea what provider(s) support complex dynamic DNAME records? It may be safer/easier to only support basic records to stat.

Reply:
You're right, no provider support `DNAME` dynamic records right now.
I've removed this part.

BTW NS1 support for dynamic records seems to be broken when the record type is anything other than `A` / `AAAA`.

Reply:
Yeah. `dynamic` CNAME is pretty much alpha atm. It was tested some back when it was written, but very lightly and it needs some more attention/time 😦